.github/CODEOWNERS

```diff
@@ -231,36 +240,44 @@
 
 # PRLabel: %Image Analysis
 # ServiceLabel: %Image Analysis
-/sdk/vision/ai-vision-image-analysis-rest/ @rhurey @dargilco 
+/sdk/vision/ai-vision-image-analysis-rest/ @rhurey @dargilco
 
 # PRLabel: %Schema Registry
+/sdk/schemaregistry/ @minhanh-phan @deyaaeldeen
+
 # ServiceLabel: %Schema Registry
 # AzureSdkOwners: @minhanh-phan
-/sdk/schemaregistry/ @minhanh-phan @deyaaeldeen 
+# ServiceOwners: @aschhabria
 
 # PRLabel: %Cognitive - Form Recognizer
+/sdk/formrecognizer/ @HarshaNalluru @jeremymeng
+
 # ServiceLabel: %Cognitive - Form Recognizer
 # AzureSdkOwners: @HarshaNalluru
-/sdk/formrecognizer/ @HarshaNalluru @jeremymeng
 
 # PRLabel: %Cognitive - Metrics Advisor
+/sdk/metricsadvisor/ @KarishmaGhiya @jeremymeng
+
 # ServiceLabel: %Cognitive - Metrics Advisor
 # AzureSdkOwners: @KarishmaGhiya
-/sdk/metricsadvisor/ @KarishmaGhiya @jeremymeng
 
 # PRLabel: %Notification Hub
+/sdk/notificationhubs/notification-hubs @mpodwysocki @jeremymeng
+
```

Comment:
@mpodwysocki is there a service owner for notification hub?

Reply:
Yes, lomagdal2

Reply:
Added @lomagdal2